bazel/astore/defs.bzl

```diff
@@ -141,7 +145,7 @@ def astore_download_and_extract(ctx, digest, stripPrefix, path = None, uid = Non
         f,
         "--overwrite",
     ]
-    res = ctx.execute(enkit_args, timeout = 60)
+    res = ctx.execute(enkit_args, timeout = 600)  # better to be slow than to fail
```

Comment:
nit: make it a parameter defaulting to 600?

Reply:
I made it an attribute.